creator-node/package.json

```diff
@@ -27,7 +27,7 @@
     "bl": "^4.1.0",
     "body-parser": "^1.18.3",
     "buffer": "5.4.2",
-    "bull": "^3.22.8",
+    "bull": "4.8.2",
```

Comment:
this will affect all other CN bull queues - since we dont have sufficient alerting, we'll want to let this soak on stage for a bit and go through other usages to make sure they all look good

i know you won't have context there so i'll do it / we can work together

did you confirm if there are any breaking changes from v3 to v4? would be hesitant to rely on docs haha

Reply:
I tested locally and looked through all the release tags between these versions and saw only 1 breaking change (it was for rate-limiting so it didn't look applicable). still makes sense to let it soak since it affects other queues like you said 👍